### PR TITLE
Entries: Document $.fn.labels

### LINKED ISSUE #270: Document $.fn.labels
We need to add an entry for the new method `$.fn.labels` which attempts to mimic the `control.labels` property from webkit browsers and extends it to work on document fragments as well. This method of course only works on a single element and like other jQuery methods will use the first element in a collection if a collection is passed. It returns a jQuery Collection of all of the labels present which are associated with the input, or an empty collection if there are none.

### PR DESCRIPTION
**Pull request by jzaefferer**
Fixes #270

### CODE REVIEW

File: entries/labels.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<entry name="labels" type="method" return="jQuery">
+	<title>.labels()</title>
+	<desc>Finds all label elements associated with the first selected element.</desc>
+	<longdesc>
+		<p>This can be used to find all the <code>&lt;label&gt;</code> elements associated with an <code>&lt;input&gt;</code> element. The association can be through nesting, where the label is an ancestor of the input, or through the <code>for</code> attribute on the label, pointing at the <code>id</code> attribute of the input. If no labels are associated with the given element, an empty jQuery object is returned.</p>
+		<p>This methods mimics the native <code>labels</code> property, which isn't supported in all browsers. In addition, this method also works for document fragments.</p>
+	</longdesc>
+	<added>1.12</added>
```
Comment (scottgonzalez) [inline entries/labels.xml]: This should be inside a `<signature>`.

**Comment by scottgonzalez** [comment]
Docs look good, just one small change to the structure.

**Comment by arschmitz** [comment]
LGTM

**Comment by jzaefferer** [comment]
Updated

**Comment by jzaefferer** [comment]
Landed d07a2f4 in 1-12, also fixed the position signature in b5d9008.

**Pull request closed without merging**